### ISSUE TITLE
create "always open" pickerInput

### ISSUE THREAD

**Issue by aclong** [dreamRs/shinyWidgets#486]
Is it possible to create an "always open" picker.

I have found and like multiinput() but don't need the second column showing what has been picked.

I want to be able to paste these columns together to create something like a "finder" window.

I thought this could be accomplished by placing pickerinputs next to one another, with an automated setting for how many rows they display at a time but I cannot work out how to get them to start off and stay open (like multiinput() can).

**Comment by pvictor** [comment]
Hello,

No that option doesn't exist for `pickerInput()`, you can look at `virtualSelectInput()` in package [shinyvs](https://github.com/dreamRs/shinyvs) where it's possible:

```r
library(shiny)
library(shinyvs)

ui <- fluidPage(
  tags$h2("Virtual Select"),
  
  virtualSelectInput(
    inputId = "single",
    label = "Single select :",
    choices = setNames(month.abb, month.name),
    search = TRUE,
    keepAlwaysOpen = TRUE
  ),
  verbatimTextOutput("res_single")
)

server <- function(input, output, session) {
  output$res_single <- renderPrint(input$single)
}

if (interactive())
  shinyApp(ui, server)
```

**Comment by pvictor** [comment]
`virtualSelectInput()` is now available in shinyWidgets 0.7.0 on CRAN.
